### DISCUSSION TITLE
Last.fm Integration

### DISCUSSION THREAD

**Discussion by ghost** [Suplanus/Cuterdio#178] [Ideas]
Please add a last.fm integration to "scrobble" (= transfer) heard songs.

API docs: https://www.last.fm/api/scrobbling

Yes I know, last.fm seems to be dead, but I have been scrobbling my music for almost 15 years. And that's also the reason why I use Spotify instead of Apple Music, because they have last.fm integrated ;)

**Comment by Suplanus** [comment]
Thanks for the request.  
Good idea. I will have a look to the API!

**Comment by costamsm** [comment]
I second the integration with Last.fm support. Would be great!

**Comment by fluteds** [comment]
Sorry to revive an old discussion but I would also love last.fm support. Is this being worked on or have I missed something in the docs?
